src/cli/utils.ts

```diff
@@ -341,3 +345,36 @@ export function getTagsFromOptions(
 ): { name: string; value: string }[] {
   return parseTags(options.tags);
 }
+
+export function currencyFromOptions<
+  O extends GlobalOptions & { currency?: string },
+>(options: O): Currency | undefined {
+  const currency = options.currency?.toLowerCase();
+
+  if (currency !== undefined && !isCurrency(currency)) {
```

Comment:
Nit: you can save clients of isCurrency some headaches by relaxing its argument type constraint to accept "unknown" or at least `| undefined`.